Update changelog for 1.1.3 (#4421)
xournalpp/xournalpp: CHANGELOG.md

## 1.1.3~dev (Unreleased)

* Fixed a crash that occurs when trying to add/edit/test LaTeX if LaTeX is not found (#4334, #4345)

> * Fixed touch enabling on quit, preventing a crash in some cases (#4343, #4373)

Reviewer: This line is confusing to me, I think this might be clearer. Not sure if it's accurate though.
```suggestion
* Fixed a crash that could occur when quitting when using a touchscreen device (#4343, #4373)
```

Author: If you want to have it more accurate: Maybe like this:
Fixed a crash that could occur when quitting when using a touchscreen device in case internal Hand Recognition is enabled in the preferences (#4343, #4373)
It sounds complicated though.

> * Fixed a bug that closed text fields immediately after creating them with the middle mouse button (#3983, #4309)

Reviewer: This fix affects more than just middle mouse:
```suggestion
* Fixed a bug that closed text fields immediately after creating them with a secondary mouse or stylus button (#3983, #4309)
```

Author: I'm fine with the suggestion.
